### DISCUSSION TITLE
Comunidade front-end em Curitiba

### DISCUSSION THREAD

**Discussion by pcesarteixeira** [frontendbr/forum#1333] [Discussão]
Olá pessoal, resido em Curitiba, e uma coisa que sinto muita falta aqui é de uma comunidade mais ativa similar a que tem em São Paulo e Rio de Janeiro. Existem bastantes empresas de tecnologia aqui que estão fazendo muitas coisas legais, temos alguns pequenos grupos de React, e uma pequena movimentação com VueJS começando, mas fica por isso mesmo.

Do pessoal que acompanha o fórum aqui, alguém é de Curitiba? Apareçam e vamos desenvolver isso ai e se conhecer. :smile:

**Comment by wpbarcelos** [comment]
Eu sou de Vitória, ES, estou no mesmo barco, mas as coisas estão melhorando, agora tem um evento DevES, ao menos 1x por ano. rsrsrs

**Comment by nandomoreirame** [comment]
Eu sou, e sim, Curitiba é bem parada mesmo.

Alguns anos atrás eu fui mantenedor do @femug aqui em Curitiba, o @femug-cwb.... tivemos vários encontros legais mas infelizmente a coisa foi esfriando e está parada agora.

Eu gostaria de ter me mantido mais ativo mas cansei de tentar, podemos unir forças novamente @pcesarteixeira e quem sabe dessa vez engrena!

@LFeh tem algumas dicas pra gente?

/bin/end

**Comment by evelew** [comment]
Eu sou de Curitiba também e realmente é muito parado.

**Comment by pcesarteixeira** [comment]
Acho que vale a tentativa @nandomoreirame. Tentar engajar tanto desenvolvedores quanto empresas.

**Comment by nandomoreirame** [comment]
Precisamos bolar umas ideias para uns projetos então @pcesarteixeira 
o que tinhamos antes eram:

- Meetups mensais com envio de newsletters informativas sobre os assuntos abordados.
- Tentamos montar um repo com os encontros https://github.com/femug-cwb/encontros
- Temos uma landing page para captar emails e enviar a newsletter: http://femug-cwb.github.io/

é isso, sem muita coisa afinal.

Legal se pudessemos montar mais projetos além do femug genérico.
Algo com mais nicho como **meetup-cwb-css**, **meetup-cwb-js**, **meetup-cwb-html** ... talvez o conteúdo fique mais qualitativo ao invés de quantitativo... sei lá, ideias?

@matheus-vieira @lidia-freitas e @fdaciuk estão como mantenedor do grupo também, ideias?

`/sbin/thanks`

**Comment by nandomoreirame** [comment]
@fdaciuk tava com um projeto de encontros meetups do @training-center eu não consegui participar, ainda existe xára?

**Comment by pcesarteixeira** [comment]
Show de bola, já tem algo, não vai ser preciso começar do zero.

Ideal seria a gente criar um grupo no Slack para desenvolver melhor essas ideias, e ver quem do pessoal que já teve algum contato com o @femug-cwb ainda está na ativa.

**Comment by nandomoreirame** [comment]
Ah, já tem um grupo no slack tbm (https://femug-cwb.slack.com/), mas nem sei mais como está o link de invite, se tem que instalar novamente...

**Comment by theandersonn** [comment]
Aqui no Rio está meio parado. Tem alguns meetups, mas no geral é difícil encontrar locais para a realização.

**Comment by matheusnunes** [comment]
Bora marcar um chopp

**Comment by matheus-rosin** [comment]
Sou de Curitiba também. O negócio aqui é tão parado que mal e mal são vistas vagas de emprego para devs, se comparado a São Paulo, Rio, Porto Alegre, Floripa, Nárnia... rs

Em conversa com um rapaz hoje, descobri o meetup de React; última edição não deu nem 100 pessoas :/

**Comment by felipefialho** [comment]
@nandomoreirame Tudo bem?

No Front in Floripa, tive uma conversa bem legal com uma galera de Blumenau extremamente engajada em fazer a comunidade da cidade crescer e criar corpo. 

Algumas coisas que disse pra eles:

- Comunidade precisa de pessoas
- Tudo começa pequeno e as pessoas vão se engajando 
- Eventos costumam melhorar o pensamento das empresas e mesmo a qualidade de devs na região, coloca isso na cabeça da galera que a coisa começa a ficar orgânica :)
- Tenta conversar com empresas da região para conseguir apoio de espaço e coffee
- Divulga bastante para atrair gente, mas se tiver organizadores e uma pessoa de fora, você já tem um meetup

Qualquer coisa, tamo ai!

**Comment by gustavofsantos** [comment]
Uma coisa que acho que seria massa ter em Curitiba é um evento tipo a BrazilJS, tipo um "cwb.js" da vida. Eu topo ajudar a fazer dar certo

**Comment by matheus-rosin** [comment]
Acredito que uma forma de tornar o grupo mais movimentado seria fazer algo "agnóstico", i.e., de qualquer coisa do mundo JS (ou quem sabe de outras línguas).
Por exemplo, a meetup de React talvez tenha pouca gente por ser só de React. Mas e se for falar de React, Angular, Vue, Cycle, Webpack, Node, VS Code, GraphQL etc.? Vai ter mais gente interessada; e também mais gente que antes selecionava aonde ia pelo assunto, mas que agora vai encontrar outras pessoas com outros interesses por causa da pluralidade.

**Comment by pcesarteixeira** [comment]
Acho que o caminho é fazer algo mais abrangente mesmo. Qualquer coisa mais focada acho que pode dar certo no inicio, mas vai ficar complicado de expandir. Como podemos nos organizar para começar a trabalhar isso?

**Comment by matheus-rosin** [comment]
Eu vou me envolver pela primeira vez com isso, então não sei quase nada rss mas posso ajudar como puder

**Comment by pcesarteixeira** [comment]
Temos dois caminhos para seguir. Ou a gente aproveita o que o @nandomoreirame já tem com o FEMUG e se organiza para dar continuidade, ou a gente tenta algo novo como **cwb.js**, essa última opção talvez deixe mais claro o objetivo e atraia mais pessoas.

O que acham melhor seguir?

**Comment by matheus-rosin** [comment]
Imagino que pontuar coisas baseado no que recolhemos aqui e no que vocês já sabem possa ajudar.

Por exemplo:

>> Tenta conversar com empresas da região para conseguir apoio de espaço e coffee

Daí fazemos uma lista de potenciais empresas.

Vamos fazendo assim com tudo que der, para a coisa toda ficar clara tanto para nós quanto para as empresas e devs. Assim, com razão e propósito bem definidos, aumenta a vontade de fazer a coisa acontecer e, consequentemente, o engajamento.

Dá para fazer uma página simples para dizer onde encontrar os eventos, quais são as empresas parceiras etc., como com o código: tudo bem documentadinho rss

**Comment by nandomoreirame** [comment]
Fala galera, que bom que vcs estão animados hein!! 🎉

Vamos lá... então...

Minha ideia era fazer pequenas raminificações do mundo front-end, e usar o @FEMUG-CWB como base geral e ir criando mini eventos de baixo do guarda-chuva com as tecnologias, assim atrair todo mundo.

Como o @matheus-rosin falou, se fizer um grupo só de React esse grupo ficará pequeno, se for de JS em geral vai aumentando, mas se for de front-end em geral é maior ainda.

Como o @LFeh falou **Comunidade precisa de pessoas**. Sem as pessoas o FEMUG morreu e qualquer coisa que a gente tentar fazer vai esfriar novamente.

@matheus-rosin vou contar um pouco das nossas experiencias e frustrações com o FEMUG e as empresas aqui de Curitiba.

A ideia: **_Tentar conversar com empresas da região para conseguir apoio de espaço e coffee_** é super válida, mas se não tivermos comprovações de pessoas e projetos as empresas não engajam. A gente fez exatamente isso mas as frustrações foram muitas pq não tinhamos como provar que o evento daria algum retorno para a empresa. Antes disso temos que colocar pra rodar, um exemplo clássico seria: _Tentar buscar um investimento anjo sem ter tirado o projeto da startup do papel._ Saca? hahahaha

Então minha sugestão seria:

> 1. Planejar os projetos, quais meetups iremos fazer, quais serão as datas, todo planejamento que pudermos criar é válido. Nesse item podemos fazer pesquisas com a comunidade para saber quais os temas são mais relevantes para a maioria, ai teremos um menor risco de falhar.
> 2. Começar a divulgar, de preferência sempre com 1 mês de antecedencia, com o planejamento feito teremos todos os meetups de pelo menos 4 a 6 meses para frente.
> 4. Juntar dados das pesquisas e de todo planejamento feito dos meetups para buscar empresas que nos ajudariam a fazer os encontros, lugares e coffee.
> 5. Continuar com os planejamentos dos próximos 6 meses de meetups.

O grande problema do nosso grupo que organizava o FEMUG-CWB era o planejamento, não conseguimos planejar com antecedencia as coisas, creio que por falta de tempo dos envolvidos.

Eu encontrei o app para invite de novos membros no Slack: https://femug-cwb.herokuapp.com/

Podemos levar a diante o grupo, quem quiser participar da organização só me chamar no Telegram: https://t.me/oseunando

`sh /bin/end`

**Comment by fdaciuk** [comment]
> @fdaciuk tava com um projeto de encontros meetups do @training-center eu não consegui participar, ainda existe xára?

Fizemos alguns encontros na Pipefy, mas foram poucos (uns 3 ou 4 se não me engano), e, com no máximo umas 10 pessoas.

Aqui é realmente difícil fazer um meetup maior que isso, não sei se por falta de costume, ou qual é, de fato, o problema.

O meetup de React sempre vai bastante gente. Participei de alguns sempre foram mais de 50 pessoas em todos os encontros. Para Curitiba, esse é um número bem considerável.

Têm muitos eventos de software livre na cidade, e esses normalmente são cheios também. Seria legal tentar trocar uma ideia com essa galera que organiza esses eventos de SL na cidade pra aprender um pouco :)

Mas acho que a ideia é essa mesmo: começar divulgando pequenos meetups, ainda que vá poucas pessoas no início, mas mantendo uma constância. Foi assim que o meetup de React cresceu.

A Pipefy é uma empresa super aberta a projetos de comunidade, acho que vale a pena tentar trocar uma ideia com eles pra tentarmos organizar alguns meetups lá :)

No mais, o mais difícil é manter a constância. Fazer um a cada 15 dias ou 1 por mês acho que seria o suficiente, mas precisa ser constante, senão não vai pra frente mesmo =)

**Comment by matheus-vieira** [comment]
Acho bem interessante a ideia!

Tentei ajudar o @nandomoreirame no femug e vi a dificuldade.

Se precisarem de ajuda podem falar comigo.

Não sou um grande conhecedor do mundo front-end mas posso dar meus pitacos.

**Comment by pcesarteixeira** [comment]
Quem tiver interesse em ajudar a organizar tudo isso e também ao pessoal que queira participar desde já, pode entrar no Slack que o @nandomoreirame postou acima, acredito que a partir de agora a discussão se desenrolará por lá. :smile: 

> Eu encontrei o app para invite de novos membros no Slack: https://femug-cwb.herokuapp.com/

**Comment by rafaelbezerra195** [comment]
Fala pessoal, 

Como o @fdaciuk falou, já houve alguns encontros do TC lá no Pipefy e foram bem legais. 
Os meetups de React foram mais constantes, acho q o último que rolou foi no meio de dezembro, inclusive eles fazem uma live no youtube, para quem não conseguiu ir no evento.

É bom aproveitar o começo para engajar a galera, tb não tenho muita experiência mas topo participar. 

Se quiserem fazer mais informal primeiro, como o @matheusnunes falou, pode ser um Choop :)

https://www.meetup.com/pt-BR/ReactJS-CWB/

**Comment by pcesarteixeira** [comment]
Só para deixar registrado aqui pessoal, criamos um grupo no Telegram para facilitar a comunicação, então quem tiver interesse seria bem legal aparecer por lá.

Link do grupo: https://t.me/femugcwb

**Comment by fdaciuk** [comment]
Boa! Telegram é bem melhor :D

**Comment by evertonthepaula** [comment]
1 - Quando cheguei aqui na Objective (estamos no andar de cima da Pipefy @fdaciuk :p ) eles já faziam Talks mensais, toda primeira semana do mês, acredito que este poderia ser um primeiro passo para acostumar as pessoas as Talks mensais, afinal  "Se você quer ajeitar o mundo, comece arrumando seu quarto".

2 - A sala grande sempre fica cheia o que inviabiliza trazer pessoas de fora, mas E SE fosse possível que uma vez por semana empresas diferentes pudessem usar o mesmo canal de lives para apresentar essas talks para o mundo? Algo como um CWBDevTalks! 

3 - Agora com os Devs convencidos do poder de falar, e com as empresas já participando do processo poderíamos começar a marcar encontros nestes lugares aos finais de semana para resolver problemas reais de software livre :D 

4 - Comunidade forte, hora de Fazer a CWBJSpirit (Nome baseado na Shinobi Spirit :smile_cat: ).

**Comment by evertonthepaula** [comment]
Segue um link para ajudar:

https://www.meetup.com/pt-BR/curitibajs/

**Comment by nillswe** [comment]
Eai galera, 
Moro de Curitiba tbm.  Não tenho mta experiência com comunidade, a primeira que participei foi uma que a pipefy fez por aqui, mas gostaria de ser mais ativo.  Se a galera animar alguma coisa eu me disponibilizo pra dar uma força (da maneira que eu puder) e a participar do evento.

**Comment by marcosdsdba** [comment]
Posso ajudar(dentro da minha disponibilidade), penso lecionando ReactJS , KnockoutJS (do básico ao avançado) com backend Java, Python ou PHP. É só combinar !!!

**Comment by danilosilvadev** [comment]
Brother todo sábado de manhã to indo pro Liquori com um amigo programar react native, se quiser cola lá, tamo fazendo apps pro portifolio, internet até q é boa, nunca ta muito cheio e tem uns bons cafés tbm, ainda fica perto do shopping hauer pra um happy hour após o coding haha. @pcesarteixeira

**Comment by pcesarteixeira** [comment]
> Brother todo sábado de manhã to indo pro Liquori com um amigo programar react native, se quiser cola lá, tamo fazendo apps pro portifolio, internet até q é boa, nunca ta muito cheio e tem uns bons cafés tbm, ainda fica perto do shopping hauer pra um happy hour após o coding haha. @pcesarteixeira

Opa! Bora rapaz.

**Comment by danilosilvadev** [comment]
@pcesarteixeira 
Liquori Caffè Gourmet, R. Comendador Araújo, 534, 10 AM no sábado. Quem quiser só me procurar lá, o projeto é open source entao de boa dar umas olhadas lá e tals, tiver dúvidas dos seus projetos pessoais tbm posso ajudar no q tiver de tempo disponível haha.

Aqui o repo do projeto https://github.com/chickenandburritos/worthit
É um simples todolist com rewards.

**Comment by evertonthepaula** [comment]
Evento em organização para Março

https://www.meetup.com/pt-BR/curitibajs/events/258524373/?rv=ea1_v2&_xtd=gatlbWFpbF9jbGlja9oAJDMxM2IyYzJmLWIxZWYtNDVhNC05M2Q1LTVlNTc3M2Y3MTI3Yw

**Comment by IuryChagas** [comment]
> Evento em organização para Março
> 
> https://www.meetup.com/pt-BR/curitibajs/events/258524373/?rv=ea1_v2&_xtd=gatlbWFpbF9jbGlja9oAJDMxM2IyYzJmLWIxZWYtNDVhNC05M2Q1LTVlNTc3M2Y3MTI3Yw

\o/ Woow.. até que enfim poder se aproximar da comunidade local!

**Comment by trickstival** [comment]
Cheguem lá no meetup gente! Ainda está aberto pra mandar palestras!

**Comment by evertonthepaula** [comment]
https://www.meetup.com/pt-BR/ReactJS-CWB/events/258726772/?rv=ea1_v2&_xtd=gatlbWFpbF9jbGlja9oAJDQ0YTc0NWI0LWFjNDgtNGJhNS05MzNmLTg4MWIwODYxZmM4Ng

**Comment by llvasconcellos** [comment]
Morando em curitiba. +1

**Comment by evertonthepaula** [comment]
https://github.com/braziljs/mascote/issues/1

Vamos ajudar o Pessoal da BrazilJs a escolher seu mascote, afinal CWBJS nós já sabemos que será a capivara né? kkkkk

**Comment by nillswe** [comment]
> [braziljs/mascote#1](https://github.com/braziljs/mascote/issues/1)
> 
> Vamos ajudar o Pessoal da BrazilJs a escolher seu mascote, afinal CWBJS nós já sabemos que será a capivara né? kkkkk

Capivara +1 kk

**Comment by nillswe** [comment]
> Brother todo sábado de manhã to indo pro Liquori com um amigo programar react native, se quiser cola lá, tamo fazendo apps pro portifolio, internet até q é boa, nunca ta muito cheio e tem uns bons cafés tbm, ainda fica perto do shopping hauer pra um happy hour após o coding haha. @pcesarteixeira

Eai Brother, esse sábado vc vai fazer esse rolê? To querendo dar uma olhada em react native e trocar umas ideas sobre.

**Comment by gustavofsantos** [comment]
@alexgwolff Curitiba TI

**Comment by adamfoerster** [comment]
@alexgwolff uma boa oportunidade de networking é participar dos meetups locais. Sou um dos organizadores do [Frontend Curitiba](https://meetup.com/FrontEnd-Curitiba/), mas Curitiba também tem o Meetup de React e o Curitiba.js

**Comment by fdaciuk** [comment]
@rlechetaudemy aqui: https://github.com/frontendbr/vagas

**Comment by evertonthepaula** [comment]
Galera tem esse Mettup querendo voltar a vida! 

https://www.meetup.com/pt-BR/Vue-Curitiba/

**Comment by evertonthepaula** [comment]
É Evento que a gente quer? É evento que a gente vai ter <3 

https://www.meetup.com/pt-BR/Vue-Curitiba/events/261228753/

**Comment by gustavogusss** [comment]
Irado Everton! Já confirmei. Tem 44 confirmados, já é alguma coisa! Espero aprender muito e fazer um bom network lá, pq to precisando xp

**Comment by evertonthepaula** [comment]
> Irado Everton! Já confirmei. Tem 44 confirmados, já é alguma coisa! Espero aprender muito e fazer um bom network lá, pq to precisando xp

OPA! Acredito que vai ter bem mais gente interessada até o evento e o grupo estava meio parado também.

**Comment by JHBitencourt** [comment]
Acho que se passaram 3 anos e nada mudou, não encontro nada de meetups de devs em Curitiba hahah

**Reply by felquis** [reply]
Se antes da pandemia já não tinha, imagino depois de uma pandemia, todo mundo traumatizado e com razão. 
Talvez voce devesse começar a puxar o meetup, encontrar uma escritório parceiro que aceita dar o espaço. E pau na lata meu amigo.

**Comment by williamkoller** [comment]
Pessoal tem algum evento de JavaScript em Curitiba?
